Maquetado/scss/tech.scss

```diff
@@ -0,0 +1,34 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  font-size: $text-small;
+  height: 320px;
+  padding: 20px 30px;
+  text-align: center;
+  width: 220px;
+
+  img {
+    height: $tech-logo-size;
+  }
+
+  h2 {
+    font-size: $text-medium;
+    margin-top: 20px;
+  }
+
+  p {
+    line-height: $text-small + 8px;
+    margin-top: 20px;
+  }
+}
+
+.tech-border {
+  border: solid 1px $green;
+  border-radius: 5px;
+  height: 320px;
+  margin: 50px; //Just for visualization
```

Comment:
add this to the `.tech` for the moment and move all this properties to the `::before`